cmd/config.go

```diff
@@ -0,0 +1,32 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/joho/godotenv"
+	"github.com/sirupsen/logrus"
+)
+
+type Config struct {
+	dbPath string
+	port   string
+}
+
+func loadConfig() (*Config, error) {
+	const op = "main.loadConfig"
+	err := godotenv.Load()
+	if err != nil {
+		return nil, fmt.Errorf("Не удалось загрузить переменные окружения из файла .env. %s:%v", op, err)
```

Comment:
- Комментарии лучше не английском
- Пробелы для лучшего чтения
- У меня Goland ругается на предложения с большой буквы, так как при склейке ошибок будет текст не очень
- Для обёртывания ошибок нужно использовать `%w`, а для вывода `%v`

```suggestion
		return nil, fmt.Errorf("не удалось загрузить переменные окружения из файла .env. %s: %w", op, err)
```